storage/move_packages.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+"""
+Automate Package Moving
+
+Helper script to automate moving packages in the Archivematica Storage Service.
+"""
+
+from __future__ import print_function
+import argparse
+import json
+import logging
+import logging.config  # Has to be imported separately
+import os
+import requests
+from six.moves import configparser
+import sys
+
+# This project
+from common import utils
+
+THIS_DIR = os.path.abspath(os.path.dirname(__file__))
+sys.path.append(THIS_DIR)
+
+LOGGER = logging.getLogger('storage')
+
+CONFIG_FILE = None
+
+
+def get_setting(setting, default=None):
+    return utils.get_setting(CONFIG_FILE, 'storage', setting, default)
+
+
+def setup(config_file):
+    global CONFIG_FILE
+    CONFIG_FILE = config_file
+
+    # Configure logging
+    default_logfile = os.path.join(THIS_DIR, 'automate-storage.log')
+    logfile = get_setting('logfile', default_logfile)
+    utils.configure_logging(logfile)
+
+
+def get_first_eligible_package_in_location(ss_url, location_uuid):
+    """
+    Get first package in a location that has a status of either UPLOADED or MOVING.
+
+    :param str ss_url: Storage service URL
+    :param str location_uuid: UUID of location to fetch package details from
+    :returns: Dict containing package details or None if none found
+    """
+    get_url = "%s/api/v2/file/" % (ss_url)
+
+    # Specify order so what query returns is consistent
+    params = [
+        ("current_location__uuid", location_uuid),
+        ("status__in", "UPLOADED"),
+        ("status__in", "MOVING"),
+        ("order_by", "uuid")]
+
+    result = utils.call_url_json(get_url, params, LOGGER)
+    if 'objects' in result and len(result['objects']):
+        return result['objects'][0]
+    else:
+        return None
+
+
+def move_to_location(ss_url, package_uuid, location_uuid):
+    """
+    Send request to move package to another location.
+
+    :param str ss_url: Storage service URL
+    :param str package_uuid: UUID of package to move
+    :param str location_uuid: UUID of location to move package to
+    :returns: Dict representing JSON response.
+    """
+    LOGGER.info("Moving package %s to location %s", package_uuid, location_uuid)
+
+    post_url = '%s/api/v2/file/%s/move/' % (ss_url, package_uuid)
+    post_body = json.dumps({'location_uuid': location_uuid})
```

Comment:
requests is actually smart enough to encode JSON itself, so you don't need to stringify it. http://docs.python-requests.org/en/master/user/quickstart/#more-complicated-post-requests